utils/config/daos_server.yml

```diff
@@ -601,3 +601,13 @@
 #
 #  bdev_auto_faulty:
 #    enable: false
+#
+## Override the default file transfer mechanism for dmg support collect-log by supplying
+## the path to an alternative script or binary that will be used to copy log off of the servers.
+## Note that the --transfer-args flag to the collect-log command may be used to supply
+## extra runtime arguments used by the copy tool (e.g. a cloud bucket name, etc.)
```

Comment:
I'd suggest moving this documentation section directly above `file_transfer_exec`.  Above the `support_config` section you could indicate that this is a configuration for support-related operations.

Reply:
Not sure what you mean. That's where it's currently located.

Reply:
I mean that these documentation lines are specific to `file_transfer_exec`, not `support_config` overall. Right now this is the only configurable option in `support_config`, but that may not always be the case.

Reply:
done